api/api.go

```diff
@@ -147,3 +170,47 @@ func getSegmentNamesInUse(conditions []dtos.ConditionDTO) map[string]bool {
 	return segmentNames
 
 }
+
+// Get info for single segment
+func (binding *SplitioAPIBinding) getSegment(segmentName string) (dtos.SegmentChangesDTO, error) {
+	path := "segmentChanges"
+	segment := dtos.SegmentChangesDTO{}
+	allChanges, since, err := binding.getAllChanges(fmt.Sprintf("%s/%s", path, segmentName))
+	if err != nil {
+		return segment, err
+	}
+
+	addedMap := map[string]bool{}
+	for _, changes := range allChanges {
+		var SegmentChanges dtos.SegmentChangesDTO
+		err = mapstructure.Decode(changes, &SegmentChanges)
+		if err != nil {
+			err = fmt.Errorf("error when decode data to segment: %s", err)
+			return segment, err
+		}
+
+		for _, add := range SegmentChanges.Added {
```

Comment:
isn't the value of the `add` variable an id?  Should we name that variable `id` instead of `add`?

Reply:
yeah it will be `shopper_id` or `visitor_guid`, I will update that.